File: ComputerBroker/src/main/java/com/marbl/domain/WorkPerformedInfo.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.marbl.domain;

import java.math.BigDecimal;

/**
 *
 * @author Bas
 */
public class WorkPerformedInfo {
    
    private String description;
    private BigDecimal price;
    
    public WorkPerformedInfo(String description){
        this.description = description;
    }
    
    public void setPrice(BigDecimal price){
        this.price = price;
    }
        
    public BigDecimal getPrice(){
        return price;
    }
    
    public String getDescription(){
        return description;
    }
}

File: ReparatieSysteem/ReparatieSysteem/Messaging/WorkPerformedInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ReparatieSysteem.Messaging
{
    class WorkPerformedInfo
    {
        private string description;

        private int price;

        public int Price
        {
            get { return price; }
            set { price = value; }
        }

        public String getDescription
        {
            get { return description; }
        }

        public WorkPerformedInfo(string description)
        {
            this.description = description;
        }
    }
}
